### PR TITLE
{tools}[dummy/] vsc-mympirun v4.0.1

### PR DESCRIPTION
**Pull request by boegel**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2570.golett.os - Linux centos linux 7.3.1611, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/da3820b9898bf22320ce573d9583dfd5 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
gligar01.gligar.os - Linux centos linux 7.3.1611, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/707e887b0e5e7516133424ea1f26400d for a full test report.

**Comment by verdurin** [comment]
Test report by @verdurin
**SUCCESS**
Build succeeded for 3 out of 3 (1 easyconfigs in this PR)
ca014.camp.thecrick.org - Linux centos linux 7.3.1611, Intel(R) Xeon(R) CPU E5-2640 v3 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/cd9628d604e4f2ff9b75754444bcb9f2 for a full test report.

File: easybuild/easyconfigs/v/vsc-mympirun/vsc-mympirun-4.0.1.eb [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+osdependencies = ['python-setuptools']
+
+# we ship something in bin/fake
+modextrapaths = {'PATH': 'bin/fake'}
+
+# don't check for import of vsc.mympirun, since that may fail if vsc-base is picked up from the OS
+options = {'modulename': 'vsc'}
+
+pyshortver = '.'.join(SYS_PYTHON_VERSION.split('.')[:2])
```
Comment (verdurin) [inline easybuild/easyconfigs/v/vsc-mympirun/vsc-mympirun-4.0.1.eb]: @boegel - isn't there a proper built-in macro for this now, so you don't need to define your own?

Comment (boegel) [inline easybuild/easyconfigs/v/vsc-mympirun/vsc-mympirun-4.0.1.eb]: Only when `Python` is listed as a dependency, not when you're using the system Python like we're doing here...

Comment (verdurin) [inline easybuild/easyconfigs/v/vsc-mympirun/vsc-mympirun-4.0.1.eb]: Got it. One day I'll catch you out.

**Comment by verdurin** [comment]
Merged, thanks @boegel

**Pull request merged**